invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/search.less

```diff
@@ -43,4 +43,20 @@
 /* Override style from webpack:///node_modules/semantic-ui-less/definitions/views/item.less line 444 */
 .ui.link.items > .item:hover {
   cursor: auto !important;
-}
\ No newline at end of file
+}
+
+.receiver {
+  margin-left: 5px;
+}
+
+.opened-by {
+  margin-right: 15px !important;
+}
+
+.meta.request-info {
+  margin-top: 10px !important;
```

Comment:
That's a nice idea!
I checked invenio-theme and I couldn't find the "padding generator" maybe I missed it, could you point me to the code if you know where it is, otherwise I can add it to invenio-theme the same way it's done for invenio-ils, should it be placed in `site.overrides` file?

Reply:
Added here https://github.com/inveniosoftware/invenio-theme/pull/254